Makefile

```diff
@@ -70,6 +70,10 @@ RPMS_TO_INSTALL = \
     tcpdump \
     xfsprogs \
     xmlrpc-c-c++ \
+    pciutils \
+    openvswitch \
```

Comment:
pciutils, unzip, nc and rpdb are required for basic debugging of any flow.
openvswitch is there since it is basic for every distribution and I don't see a reason to open a new repo just for this